app/poros/background_image.rb

```diff
@@ -0,0 +1,23 @@
+class BackgroundImage
+  attr_reader :id, :place, :image_url, :source, :author
+
+  def initialize(data, location)
+    @image_url = data[:webformatURL]
+    @source = get_source(data)
+    @author = data[:user]
+    @place = capitalize_location(location)
+  end
+
+  private
+
+  def get_source(data)
```

Comment:
This is relatively brittle because it currently only expects data and location to come from a certain image API, but that functionality can be expanded at a later date.